diff -Naur -x '*~' -x '*.orig' -x '*.rej' bash-completion-20100203/bash_completion bash-completion-20100203-disable-user-completion/bash_completion
--- bash-completion-20100203/bash_completion	2010-02-02 19:02:26.000000000 +0100
+++ bash-completion-20100203-disable-user-completion/bash_completion	2010-02-03 23:08:26.000000000 +0100
@@ -1744,9 +1744,6 @@
 fi
 unset i
 
-# source user completion file
-[[ $BASH_COMPLETION != ~/.bash_completion && -r ~/.bash_completion ]] \
-    && . ~/.bash_completion
 unset -f have
 unset UNAME USERLAND have
 
