diff --git a/bash_completion b/bash_completion
index d16b10a..537dcb8 100644
--- a/bash_completion
+++ b/bash_completion
@@ -2077,10 +2077,6 @@ if [[ -d $compat_dir && -r $compat_dir && -x $compat_dir ]]; then
 fi
 unset compat_dir i _blacklist_glob
 
-# source user completion file
-user_completion=${BASH_COMPLETION_USER_FILE:-~/.bash_completion}
-[[ ${BASH_SOURCE[0]} != $user_completion && -r $user_completion ]] \
-    && . $user_completion
 unset user_completion
 
 unset -f have
