diff -uraN bash-completion-2.11/bash_completion omv-bash-completion-2.11/bash_completion
--- bash-completion-2.11/bash_completion	2020-07-25 13:49:49.000000000 +0200
+++ omv-bash-completion-2.11/bash_completion	2020-07-25 14:17:54.143071672 +0200
@@ -2251,10 +2251,6 @@
 fi
 unset compat_dir i _blacklist_glob
 
-# source user completion file
-user_completion=${BASH_COMPLETION_USER_FILE:-~/.bash_completion}
-[[ ${BASH_SOURCE[0]} != "$user_completion" && -r $user_completion && -f $user_completion ]] &&
-    . $user_completion
 unset user_completion
 
 unset -f have
