diff -Naur bash-completion-2.1.orig/completions/rpm bash-completion-2.1/completions/rpm
--- bash-completion-2.1.orig/completions/rpm	2014-02-05 23:24:04.000000000 +0400
+++ bash-completion-2.1/completions/rpm	2014-02-05 23:25:47.000000000 +0400
@@ -8,7 +8,7 @@
         /var/log/rpmpkgs -nt /var/lib/rpm/Packages ]]; then
         # using RHL 7.2 or later - this is quicker than querying the DB
         COMPREPLY=( $( compgen -W "$( sed -ne \
-            's|^\([^[:space:]]\{1,\}\)-[^[:space:]-]\{1,\}-[^[:space:]-]\{1,\}\.rpm$|\1|p' \
+            's|^\([^[:space:]]\{1,\}\)-[^[:space:]-]\{1,\}-[^[:space:]-]\{1,\}-[^[:space:]-]\{1,\}|\1|p' \
             /var/log/rpmpkgs )" -- "$cur" ) )
     elif type rpmqpack &>/dev/null ; then
         # SUSE's rpmqpack is faster than rpm -qa
