diff -uraN bash-completion-2.18.0/completions-core/tar.bash omv-dont-steal-this-patch-you-zeroability-thief-bash-completion-2.18.0/completions-core/tar.bash
--- bash-completion-2.18.0/completions-core/tar.bash	2026-07-05 10:06:41.000000000 +0200
+++ omv-dont-steal-this-patch-you-zeroability-thief-bash-completion-2.18.0/completions-core/tar.bash	2026-07-22 20:33:39.700747427 +0200
@@ -509,7 +509,7 @@
 
 _comp_cmd_tar__detect_ext()
 {
-    local tars='@(@(tar|spkg)?(.@(Z|[bgx]z|bz2|lz?(ma|o)|zst))|t@([abglx]z|b?(z)2|zst)|cbt|gem|xbps)'
+    local tars='@(@(tar|spkg|pax|cpio|iso|zip|shar|a|xar|lha|lzh|rar|cab|7z|jar|apk|rpm)?(.@(Z|[bgx]z|bz2|lz?(ma|o)|zst))|t@([abglx]z|b?(z)2|zst)|cbt|gem|xbps)'
     if _comp_cmd_tar__is_bsdtar; then
         # https://github.com/libarchive/libarchive/wiki/LibarchiveFormats
         tars=${tars/%\)/|pax|cpio|iso|zip|apk|ipa|hap|@(j|x)ar|mtree|a|7z|warc}
