diff -up bash-completion-2.12.0/bash_completion.1~ bash-completion-2.12.0/bash_completion
--- bash-completion-2.12.0/bash_completion.1~	2024-02-26 20:58:54.907673246 +0100
+++ bash-completion-2.12.0/bash_completion	2024-02-26 21:00:16.384484324 +0100
@@ -3330,16 +3330,6 @@ for _comp__init_compat_dir in "${_comp__
 done
 unset -v _comp__init_compat_dirs _comp__init_compat_dir _comp__init_file
 
-# source user completion file
-#
-# Remark: We explicitly check that $user_completion is not '/dev/null' since
-#   /dev/null may be a regular file in broken systems and can contain arbitrary
-#   garbages of suppressed command outputs.
-_comp__init_user_file=${BASH_COMPLETION_USER_FILE:-~/.bash_completion}
-[[ $_comp__init_user_file != "${BASH_SOURCE[0]}" && $_comp__init_user_file != /dev/null && -r $_comp__init_user_file && -f $_comp__init_user_file ]] &&
-    . "$_comp__init_user_file"
-unset -v _comp__init_user_file
-
 unset -f have
 unset -v have
 
